CHANGELOG.md

```diff
@@ -1,5 +1,83 @@
 # ember-cli Changelog
 
+### 2.2.0-beta.1
+
+The following changes are required if you are upgrading from the previous
```

Comment:
users with the existing style reloader addon, should remove it as we have proper style/image hot reloading now (both together may be unhappy)